web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -286,7 +287,7 @@ public static boolean startModule(Module mod, ServletContext servletContext, boo
 			// mark to delete the entire module web directory on exit
 			// this will usually only be used when an improper shutdown has occurred.
 			String folderPath = realPath + "/WEB-INF/view/module/" + mod.getModuleIdAsPath();
```

Comment:
Note that folderPath on the line above contains forward slashes, you need to get rid of them too